analyse_main.py

```diff
@@ -73,6 +74,12 @@ def output_properties_to_file(options, traj):
                 outputarraytofile("Self Diffusion Coefficient Array",self_diffusion_coefficient_calc(traj),f),
             'MSD' :
                 outputarraytofile("MSD",MSD_data_calc(traj),f),
+            'Cohesive Energy' :
+                outputSingleProperty(
+                    f,
+                    'Cohesive Energy',
+                    retrieve_cohesive_energy("coh_E.traj")
```

Comment:
Same comment as in cohesive_energy.py.

Reply:
Don't really know what comment in cohesive_energy.py you are referring to? Is it the name of the file?